common/types/loadbalancer.go

```diff
@@ -372,7 +391,12 @@ func (a *L3n4Addr) IsIPv6() bool {
 // KVStore.
 type L3n4AddrID struct {
 	L3n4Addr
-	ID ServiceID
+	ID    ServiceID
+	Slave uint16 // Number of slaves associated with this L3n4AddrID
```

Comment:
This changes the structure used in the kvstore. Is this safe? What will happen if we upgrade Cilium and the kvstore contains old services entries?

Reply:
You're right. This actually isn't used anywhere in the cilium-agent code, so I'm going to revert it. I figured it would be nice to have to add it now and then actually implement it at a later date, but if it's going to mess with the kv-store, I don't think it's worth it at this time. I'll have this removed in the next upload.